packages/protocol/engine/throughputquota/mana1/manamodels/apiutils.go

```diff
@@ -68,6 +69,17 @@ func Percentile(id identity.ID, m map[identity.ID]int64) (percentileValue float6
 	return (nBelow / float64(len(m))) * 100
 }
 
+// NeighborsAverageMana returns the average mana of the eighbors, based on provided mana vector and neighbors list.
+func NeighborsAverageMana(m map[identity.ID]int64, neighbors []*p2p.Neighbor) (avg float64) {
+	if len(m) == 0 {
+		return 0
+	}
+	for _, n := range neighbors {
+		avg += float64(m[n.Peer.ID()])
+	}
+	return avg / float64(len(m))
+}
```

Comment:
why is `neighbors` needed? we can iterate `m`, sum all mana and divide by `len(m)`